src/Form/DataTransformer/ProviderDataTransformer.php

```diff
@@ -33,11 +34,17 @@ class ProviderDataTransformer implements DataTransformerInterface, LoggerAwareIn
     protected $options;
 
     /**
-     * @param Pool   $pool
+     * @var CategoryManagerInterface
+     */
+    protected $categoryManager;
+
+    /**
+     * @param Pool $pool
      * @param string $class
-     * @param array  $options
+     * @param array $options
+     * @param CategoryManagerInterface|null $categoryManager
```

Comment:
We no longer add tautologic comments, you can remove the whole block except line 43